src/transformers/GovUKPayPaymentEventMessage.test.ts

```diff
@@ -14,18 +14,15 @@ describe('message formatter', () => {
 		'will_have_empty_space': ' some-empty-space-values ',
 		'value_omitted': '',
 		'value_included': 'some-value',
+		'boolean_string_with_spaces_true': '		TruE	',
+		'boolean_string_with_spaces_false': '		fAlSE		',
 		'boolean_string_lowercase_true': 'true',
 		'boolean_string_uppercase_true': 'TRUE',
 		'boolean_string_lowercase_false': 'false',
 		'boolean_string_uppercase_false': 'FALSE',
 		'empty_column': ''
 	}
 
-	// @ts-ignore
-	messageFromS3Csv["boolean_value"] = true
```

Comment:
We should still have a test to ensure numeric values are included as the same data type. Sending these values as strings in event_details JSON will cause issues in Ledger (either at the time of projecting transactions or when querying via the API).

Reply:
There arne't any numeric values though, the only possible values from S3 Select are strings, even with the code as is everything is a string or a boolean.

Without explicit casts (which we don't have in the SQL for S3 select) everything is a string when the source data is a CSV (this isn't true from parquet, but it is from CSV).

I'll attempt to prove this by adding some debugging, deploying it to test, and running it.

Reply:
After deploying a version with debug logging it's confirmed that the values are always strings:


Transformer incoming message: 
```json
{
  "transaction_id": "ne0e57l8k6uaolt7f6skuvr12m",
  "transaction_type": "payment",
  "event_name": "PAYMENT_STATUS_CORRECTED_TO_ERROR_BY_ADMIN",
  "event_date": "2024-12-20T11:25:00.000000Z",
  "refund_status": "unavailable",
  "refund_amount_refunded": "0",
  "refund_amount_available": "0",
  "updated_reason": "Payment set to PAYMENT_STATUS_CORRECTED_TO_ERROR_BY_ADMIN for TESTING PP-11677",
  "zendesk_ticket_id": "",
  "admin_github_id": "Jonathan Harden (jfharden)"
}
```

Transformer outgoing formatted message
```json
{
  "event_details": {
    "refund_status": "unavailable",
    "refund_amount_refunded": "0",
    "refund_amount_available": "0",
    "updated_reason": "Payment set to PAYMENT_STATUS_CORRECTED_TO_ERROR_BY_ADMIN for TESTING PP-11677",
    "admin_github_id": "Jonathan Harden (jfharden)"
  },
  "resource_external_id": "ne0e57l8k6uaolt7f6skuvr12m",
  "resource_type": "payment",
  "timestamp": "2024-12-20T11:25:00.000000Z",
  "event_type": "PAYMENT_STATUS_CORRECTED_TO_ERROR_BY_ADMIN"
}
```